packages/directory/addon/mixins/search-filter.js

```diff
@@ -17,10 +17,10 @@ export default Mixin.create({
     /**
      * @property searchResults - Search and rank through items in model when a search query is available
      */
-    searchResults: computed('directory.q', 'model', function() {
-        let items = get(this, 'model'),
-            queryString = get(this, 'directory.q');
-
-        return isEmpty(queryString) ? items : searchRecords(items, queryString, 'title');
+    searchResults: computed('directory.q', 'model.items', function() {
```

Comment:
Can we add the `Promise` type to the jsdoc?

Reply:
done